### ISSUE TITLE
binary cross entropy losss

### ISSUE THREAD

**Issue by Blubberli** [Blubberli/ambiguous-alpaca#5]
- implement binary cross entropy within the loss function script:
- use the torch.nn.functional API
- use binary_cross_entropy_with_logits: this applys sigmoid on input **and** computes the loss, thus a non-normalized input is expected
